tests/DucksNet.UnitTests/MedicineTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DucksNet.Domain.Model;
+using DucksNet.Domain.Model.Enums;
+using FluentAssertions;
+using Xunit;
+
+namespace DucksNet.UnitTests;
+
+public class MedicineTests
+{
+    [Fact]
+    public void When_CreateMedicine_WithValidAdministration_Should_Succeed()
+    {
+        string name = "Name";
+        string description = "Description";
+        double price = 1;
+        string drugAdministration = "Oral";
+        var result = Medicine.Create(name, description, price, drugAdministration);
+        result.IsSuccess.Should().BeTrue();
+        result.Value.Should().NotBeNull();
+        result.Value!.DrugAdministration.Name.Should().Be(drugAdministration);
```

Comment:
fix by ensuring DrugAdministration is not null.
`result.Value!.DrugAdministration!...`